specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -1459,255 +1618,2049 @@
+        "description": "Create a snapshot policy",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "Snapshot policy object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Snapshot Policy created",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "x-ms-examples": {
+          "SnapshotPolicies_Create": {
+            "$ref": "examples/SnapshotPolicies_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Snapshot Policy"
+        ],
+        "operationId": "snapshotPolicies_Update",
+        "description": "Patch a snapshot policy",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "Snapshot policy object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicyPatch"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "SnapshotPolicies_Update": {
+            "$ref": "examples/SnapshotPolicies_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Snapshot Policy"
+        ],
+        "operationId": "snapshotPolicies_Delete",
+        "description": "Delete snapshot policy",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted -- Create or update request accepted; operation will complete asynchronously"
+          },
+          "204": {
+            "description": "NoContent -- Resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "SnapshotPolicies_Delete": {
+            "$ref": "examples/SnapshotPolicies_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/snapshotPolicies/{snapshotPolicyName}/listVolumes": {
```

Comment:
why is a GET API named "ListVolumes" which make it seem like an action? Can it just be volumes?

Reply:
@ravbhatnagar  Good point, since this is live RP a backend change ticket has been created